src/test/java/com/faforever/api/map/MapControllerTest.java

```diff
@@ -0,0 +1,92 @@
+package com.faforever.api.map;
+
+import com.faforever.api.authentication.AuthenticationService;
+import com.faforever.api.config.FafApiProperties;
+import com.faforever.api.config.TestWebSecurityConfig;
+import com.faforever.api.player.PlayerRepository;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.ByteStreams;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.io.InputStream;
+
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.fileUpload;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(MapsController.class)
+@Import(TestWebSecurityConfig.class)
```

Comment:
`@Import` allows configuration classes to import other configuration classes. Use `@ContextConfiguration` instead.